helpers.go

```diff
@@ -1228,7 +1228,7 @@ func (ctx *e2econtext) runManualRemediation(t *testing.T, cmdctx goctx.Context,
 	out, err := cmd.CombinedOutput()
 
 	if errors.Is(cmdctx.Err(), goctx.DeadlineExceeded) {
-		t.Errorf("Command '%s' timed out", rem)
+		t.Errorf("Timed out applying manual remediation script '%s': %s\nError: %s", rem, out, err)
```

Comment:
Find a better way to relay this information or adjust the formatting so that it's more readable.

Reply:
I think the error `err` that will be printed out is from the `CombinedOutput()`, and you actually want the error from `cmdctx.Err()`, no?